src/styles/globals.css

```diff
@@ -19,6 +22,7 @@ body {
 	--border-radius: 12px;
 	--text-colour-light: white;
 	--text-colour-dark: black;
+	--text-colour-p: #6d6a6a;
```

Comment:
If it's not dark or light, it can be gray. But we cannot theme a project based on tag name